numberformat/proposed.emu

```diff
@@ -120,6 +120,7 @@
         1. Let _roundingIncrement_ be ? GetNumberOption(_options_, *"roundingIncrement"*, 1, 5000, 1).
         1. If _roundingIncrement_ is not in « 1, 2, 5, 10, 20, 25, 50, 100, 200, 250, 500, 1000, 2000, 2500, 5000 », throw a *RangeError* exception.
         1. If _roundingIncrement_ is not 1 and _numberFormat_.[[RoundingType]] is not ~fractionDigits~, throw a *RangeError* exception.
+        1. If _roundingIncrement_ and _numberFormat_.[[MaximumFractionDigits]] is not equal to _numberFormat_.[[MinimumFractionDigits]], throw a *RangeError* exception.
         1. Set _numberFormat_.[[RoundingIncrement]] to _roundingIncrement_.
```

Comment:
Hi @sffc this would be enough to solve the #81 ? Or we should further validate if values MaxFDigits/ MinFDigits before throw